docs/guide/docs/manage/configuration.md

```diff
@@ -0,0 +1,123 @@
+---
+id: configuration
+title: Configuration
+---
+
+There is currently an interface on Botpress to manage your server, bot's and module configuration. Most of the configuration is done using `JSON` files in the code editor.
+
+There are also some configurations that can be edited by using environment variables.
+
+On this page, you will learn about the Botpress global configuration, individual bot configuration, module configuration, and environment variables.
+
+# Botpress Global Config
+
+This is the main file used to configure the Botpress server. It will be created automatically when it is missing. Default values should be good when discovering Botpress, but in this page you will learn about the most common configuration you may need to change.
+
+To get more information about each individual options, check out the [comments on the configuration schema](https://github.com/botpress/botpress/blob/master/src/bp/core/config/botpress.config.ts)
+
+## HTTP Server Configuration
+
+By default, Botpress will start an HTTP server on localhost, listening to port 3000. If the configured port is already in use, it will pick the next available one. You can change these by editing `httpServer.host` and `httpServer.port`.
+
+### Exposing your bot on the internet
+
+When you are ready to expose your bot externally, you will need to change some of the server settings. The server doesn't support HTTPS connections, so you will need to set up a reverse proxy in front of it (for example: NGINX).
+
+This means that your server will still listen for connections on port 3000, but your reverse proxy will answer for queries on port 80. It's also the reverse proxy that will handle secure connections if you want to access your bot using `https`
+
+At this point, Botpress doesn't know how to access the bot from the web. You will need to edit the configuration of `httpServer.externalUrl`. Set the configuration variable to the complete host name, for example `https://bot.botpress.com`
+
+## Logs Configuration
+
+Logs are very useful to debug and understand what happens when the bot doesn't behave as expected.
+
+When you start Botpress from the binary (or using the Docker image), the bot is in `debug` mode. This means that a lot of information will be displayed in the console to understand what happens.
```

Comment:
From previous discussions with Marketing we tend to always use "chatbot" instead of "the bot".

Reply:
My I suggest to rephrase that into "your chatbots will run in debug mode resulting in a very high logs verbosity".